app/code/core/Mage/Dataflow/Model/Convert/Parser/Abstract.php

```diff
@@ -87,7 +87,7 @@ public function getBatchExportModel()
     /**
      * Retrieve Batch import model
      *
-     * @return Mage_Dataflow_Model_Import_Export
+     * @return Mage_Dataflow_Model_Import
```

Comment:
See line 95,`$object = Mage::getModel('dataflow/batch_import');` it should be:
```suggestion
     * @return Mage_Dataflow_Model_Batch_Import
```